### PR TITLE
IPBS member stations widget and styling

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Adds the IPBS Member Stations widget along with relevant styling for it

Desktop:
screencapture-ipbs-test-2020-01-29-16_07_02

Mobile:
screencapture-ipbs-test-2020-01-29-16_07_24

## Why

For #58

## Testing/Questions

Features that this PR affects:

- IPBS member stations widget

Questions that need to be answered before merging:

- [x] Is this PR targeting the correct branch in this repository?
- [ ] For the mobile version (and desktop, too, kinda), instead of being able to horizontally center the station content, I had to keep it left aligned so that the arrow transition still works as expected. Is that ok?

Steps to test this PR:

1. Add the IPBS Member Stations widget to the homepage bottom widget area
2. View it on desktop and mobile and make sure nothing funky happens with it

### CODE REVIEW

File: wp-content/themes/ipbs-largo/less/widgets.less [hunk 1]
```diff
@@ -9,4 +9,57 @@
         margin-bottom: @measure * 2;
         color: @blue;
     }
+}
+
+// member stations widget
+.widget.ipbs-member-stations {
+    margin-bottom: @measure * 2;
+    ul {
+        @media (min-width: 1000px) {
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/widgets.less:18]: Would it be better to use a `flex` layout with flex-basis on the `li` of about 380px, no flex-shrink, yes flex-grow? That way we have three-column, two-column, and one-column layouts.

🤔 but that makes borders harder.

File: wp-content/themes/ipbs-largo/less/widgets.less [hunk 2]
```diff
@@ -9,4 +9,57 @@
...
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+        }
+        li {
+            display: flex;
+            align-items: center;
+            border-bottom: 1px solid @grey;
+            padding-top: @measure;
+            padding-bottom: @measure;
+            padding-left: @measure * 2;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/widgets.less]: Can we have padding-right as well?

Screen Shot 2020-01-30 at 14 16 38

**Review by benlk — CHANGES_REQUESTED** [review]
Images need a max-width and max-height.

Screen Shot 2020-01-30 at 14 16 59
Screen Shot 2020-01-30 at 14 17 18

File: wp-content/themes/ipbs-largo/less/widgets.less (hunk 2 above)
Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/widgets.less]: @benlk If we add a right padding it'll cause the same issues with the arrow that we were seeing in #68

Comment (benlk) [reply wp-content/themes/ipbs-largo/less/widgets.less]: Here's a possible solution to the arrow problems, where the width of the element with the `.arrow` styles was moving around:

```diff
diff --git a/wp-content/themes/ipbs-largo/less/mixins.less b/wp-content/themes/ipbs-largo/less/mixins.less
index 477e1a7..edab44f 100644
--- a/wp-content/themes/ipbs-largo/less/mixins.less
+++ b/wp-content/themes/ipbs-largo/less/mixins.less
@@ -21,14 +21,19 @@
 }
 
 .arrow {
+    position: relative;
+    padding-right: 0.5rem;
     &::after {
         content: "\f345";
+        width: 1ch;
+        position: relative;
         font-family: 'dashicons';
         margin-left: 0.5rem;
         vertical-align: bottom;
-        transition: margin-left 0.3s;
+        transition: left 0.3s;
+        left: 0;
     }
     &:hover::after {
-        margin-left: 1rem;
+        left: 1ch;
     }
-}
\ No newline at end of file
+}

```

With element bounding boxes shown via pesticide.io:

hover

**Comment by joshdarby** [comment]
> Images need a max-width and max-height.

Added in https://github.com/INN/umbrella-ipbs/pull/73/commits/958dec9205906e92d59ed18b8fdff995cff259de

File: wp-content/themes/ipbs-largo/less/widgets.less (hunk 2 above)
Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/widgets.less]: This worked for the most part, except I had to change 

```
&:hover::after {
   left: 1ch;
}
```

to

```
&:hover::after {
   left: 10px;
}
```

Updated in https://github.com/INN/umbrella-ipbs/pull/73/commits/1f8ae0a5ef262595251080374e4a1f9423c5c2b9

Comment (joshdarby) [reply wp-content/themes/ipbs-largo/less/widgets.less]: It even works with `float: right;`! Good work 🤙

File: wp-content/themes/ipbs-largo/less/widgets.less [hunk 3]
```diff
@@ -9,4 +9,61 @@
...
+            display: grid;
+            grid-template-columns: repeat(3, 1fr);
+        }
+        li {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            border-bottom: 1px solid @grey;
+            padding-top: @measure;
+            padding-bottom: @measure;
```
Comment (benlk) [inline wp-content/themes/ipbs-largo/less/widgets.less:28]: With neither padding-right nor padding-left, around 1000px-1024px, the borders get cramped. If we decrease the image max-width to 100px, and put a left/right padding on these cells, then I think this looks better.

But that should not block merging of this PR.

**Review by benlk — APPROVED** [review]

**Pull request merged**